inginious/frontend/static/js/studio.js

```diff
@@ -343,6 +345,8 @@ function studio_create_new_subproblem()
 
     studio_create_from_template('#' + new_subproblem_type, new_subproblem_pid);
     studio_init_template(new_subproblem_pid, {"type": new_subproblem_type.substring(11)});
+
+    $('#tab_subproblems .card').on('show.bs.collapse',refresh_codemirror);
```

Comment:
Looks like a duplicate  of https://github.com/UCL-INGI/INGInious/pull/708/files#diff-dba29ffb384c39f13ccca8801458355aae23552265a109c2bd298ec22887decaR33 ?

Reply:
It's a double call of the function that refresh codemirror element. One is done on studio loading and another is done when a new sub problem is created.